forward_engineering/helpers/typeHelper.js

```diff
@@ -248,7 +248,7 @@ function getAdditionalProperties(data) {
 	}
 
 	if (data.additionalPropControl === 'Boolean') {
-		return data.additionalProperties || undefined;
+		return Boolean(data.additionalProperties) && undefined;
```

Comment:
I would propose a slightly more obvious option:
```suggestion
		return data.additionalProperties ?  undefined : false;
```
Also, it would be nice to have a comment with the explanation that the additionalProperties prop is omitted if it's `true` (because it's enabled by default, so it's the same), and included only if missed or `false`.